cvmfs/mountpoint.cc

```diff
@@ -212,8 +212,14 @@ bool FileSystem::CreateCache() {
     if (options_mgr_->GetValue("CVMFS_CACHE_RAM_MALLOC", &optarg)) {
       if (optarg == "arena") {
         alloc = MemoryKvStore::kMallocArena;
+      } else if (optarg == "heap") {
+        alloc = MemoryKvStore::kMallocHeap;
       }
     }
+    if (cache_bytes % 8 != 0) {
+      // Ensure 8-byte alignment for heap allocator
+      cache_bytes += 8 - (cache_bytes % 8);
+    }
     cache_bytes = max((uint64_t) 200*1024*1024, cache_bytes);
```

Comment:
Not part of this PR, but it would be better to define a constant like `kDefaultRamCacheSize`